app/models/cafe_work.rb

```diff
@@ -33,17 +33,7 @@ def send_email
     Rails.logger.info 'Mailer could not connect, rescued here'
   end
 
-  def old_profile
-    if profile_id.present?
-      if user_id == nil
-        return User.find_by(id: profile_id)
-      else
-        return user
-      end
-    else
-      return user
-    end
-  end
+  alias old_profile user
```

Comment:
@davidwessman Please verify that this is correct. I'm not sure I understand what the code is supposed to be doing.